lms/djangoapps/instructor/offline_gradecalc.py

```diff
@@ -50,9 +51,14 @@ def offline_grade_calculation(course_key):
         request.session = {}
 
         gradeset = grades.grade(student, request, course, keep_raw_scores=True)
-        gs = enc.encode(gradeset)
+        # Convert Score namedtuples to dicts:
+        for section in gradeset['totaled_scores']:
+            gradeset['totaled_scores'][section] = [score._asdict() for score in gradeset['totaled_scores'][section]]
```

Comment:
@bradenmacdonald This code contains many references to `gradeset['totaled_scores']`. I think that introducing a variable for it it would make it even more readable:

```python
totaled_scores = gradeset['totaled_scores']
for section in totaled_scores:
    totaled_scores[section] = [score._asdict() for score in totaled_scores[section]]
```